css-two-column-layout/styles.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Karla, sans-serif;
+}
+
+.header {
+  text-align: center;
+  margin-bottom: 45px;
+}
+
+p.headingText {
+  width: 400px;
+  margin: 0 auto;
+}
+
+p {
+  font-size: 9px;
+  line-height: 15px;
+}
+
+.container {
+  max-width: 600px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-fourth {
+  width: calc(200% / 4);
+  display: flex;
+  justify-content: center;
```

Comment:
If you feel a need to add `display: flex` to a column class, that usually mean you need a `row` inside your `column `